wfsim/sst.py

```diff
@@ -1281,6 +1314,50 @@ def _compute_m2_temperature(self):
 
         self._m2_fea_temperature = out
 
+    def _compute_m2_lut(self):
+        if self._m2_fea_lut is not _Invalidated:
+            return
+        from scipy.interpolate import interp1d
+        
+        num_actuators = 78
+        num_tangent = 6
+        num_axial_actuators = num_actuators - num_tangent
+        LUT_force = np.zeros(num_actuators)
+
+        #LUT_force_elevation
+        data = _fits_cache("M2_LUT_F_E.fits.gz")
+        LUT_force[:num_axial_actuators] += interp1d(data[0], data[1:])(
+            25
+        )
+
+        #LUT_force_0g_component
+        data = _fits_cache("M2_LUT_F_0.fits.gz")
+        LUT_force[:num_axial_actuators] += interp1d(data[0], data[1:])(
+            25
+        )
+
+        #LUT_force_factory_offset
+        data = _fits_cache("M2_LUT_F_F.fits.gz")
+        LUT_force[:num_axial_actuators] += interp1d(data[0], data[1:])(
+            25
+        )
+
+        #LUT_force_actuator_bias
+        data = _fits_cache("M2_LUT_F_A.fits.gz")
+        LUT_force += interp1d(data[0], data[1:])(
+            25
+        )
+
+        data = _fits_cache("M2_GT_grid.fits.gz")
+        zf, hf = data[0:2]
+
+        u0 = zf * (np.cos(self._m2_zenith_angle) - 1)
+        u0 += hf * np.sin(self._m2_zenith_angle)
+        u0 *= 1e-6  # micron -> meters
+
+        G = _fits_cache("M2_1um_grid.fits.gz")
+        self._m2_fea_lut = G.dot(LUT_force[:75]) - u0
```

Comment:
This is the only confusing part. I think we have 73 axial actuators, but instead the M2 influence matrix is size 75, so this is wrongly including 2 actuators. Not sure if you have any insight on this